README.md

```diff
@@ -0,0 +1,50 @@
+# Portofilo project
+
+This is the a portofilo project  
+
+![screenshot]('./Screenshot from 2021-10-21 03-58-06.png')
+
+## Built With
+
+
+- HTML, CSS
+- stylelint
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Live server on my local machine
+
+### Setup
+
+Open with live server on vscode
+
+### Install
+no installations
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ReAliens)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Please customize the readme file in detail on the necessary lines. For instance, instead of Author1, you can use your name. The `@githubhandle` can be replaced with your nick from GitHub, and if you don't have any acknowledgements to do you can simply delete that section of your README.